devenv/local/docker-compose/docker-compose.yml

```diff
@@ -460,74 +460,38 @@ services:
 
   # Modifies the CDK template and creates DynamoDB Tables if necessary.
   aws-setup:
-    build: ../../aws-setup
+    build:
+      context: .
+      dockerfile: ./Dockerfile.emily
+      target: aws-setup
     depends_on:
       dynamodb:
         condition: service_started
-    volumes:
-      - "../../aws-setup/initialize.py:/initialize.py"
-      - "../../../emily/cdk/cdk.out:/cdk.out"
     environment:
       - DYNAMODB_ENDPOINT=http://dynamodb:8000
-      - LOCAL_LAMBDA_PATH=/code/emily-handler/bootstrap.zip
-      - INPUT_CDK_TEMPLATE=/cdk.out/EmilyStack.template.json
-      - OUTPUT_CDK_TEMPLATE=/cdk.out/EmilyStack.devenv.template.json
-    command: python3 ./initialize.py
     networks:
       - aws-local-vpc
```

Comment:
Is this still correct, or should it be on the same network as the rest of the stuff?

Reply:
Not sure, I'll remove and try it